Regression/Checksum/benchmarks_json/scraping_filter.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
If you remove the checksum comparison this file can be removed.

Reply:
Removed